### ISSUE TITLE
Add a command to fetch log from a database

### ISSUE THREAD

**Issue by remyleone** [scaleway/scaleway-cli#1128]
## Description

At the moment we need to prepare and list logs to get the URL to download the logs.
It would be nice to have a single command that can perform all those operations in one go.

## How this functionality would be exposed

`scw rdb logs get 11111111-1111-1111-1111-111111111111 from=X to=Y -o myLogs.txt`

## Version

version     2.0.0-beta.4&#43;dev
build-date  unknown
go-version  go1.14.4
git-branch  unknown
git-commit  unknown
go-arch     amd64
go-os       darwin

**Comment by jerome-quere** [comment]
I would prefer if the command printed log on stdout instead of passing a file

**Comment by jerome-quere** [comment]
Blocked by #1162

**Comment by jerome-quere** [comment]
Because log preparation could return multiple files, it could be better to save them on disk
